src/components/Controller/Controller.tsx

```diff
@@ -331,96 +331,100 @@ const Controller = () => {
               className="rounded bg-orange-300 p-2 text-white disabled:opacity-50"
               onClick={() => serial.startReading()}
             >
-              Start reading console
+              Reading console...

```

Comment:
This is a button that when clicked starts streaming console messages, so I think the previous name makes a bit more sense, but really in the long term I will remove it as its not needed or used anymore

Reply:
Ahhhh, makes sense! Reverted this, thanks for pointing out!